go.mod

```diff
@@ -377,3 +377,7 @@ replace k8s.io/client-go => k8s.io/client-go v0.23.3
 
 // needed for fixing CVE-2020-29529
 replace github.com/hashicorp/go-slug => github.com/hashicorp/go-slug v0.5.0
+
+// needed for fixing CVE-2020-26160
+// taken from https://github.com/openshift/installer/blob/21cd5218bb58288cd7b03018b9a2513aca3a13a5/terraform/providers/ibm/go.mod
+replace github.com/dgrijalva/jwt-go v3.2.0+incompatible => github.com/golang-jwt/jwt v3.2.1+incompatible
```

Comment:
This commit seems like it could/should be a separate PR? Is there a reason it's included here?

Reply:
My only beef was tracking it against the CVE via its own card or whatever.